src/UXCapture.js

```diff
@@ -8,35 +9,49 @@ let _onMeasure;
 let _view;
 
 const UXCapture = {
+	_clearMarksAndMeasures: () => {
+		window.performance.clearMarks();
+		window.performance.clearMeasures();
+	},
+
 	/**
 	 * Sets `onMark` and `onMeasure` callbacks on UXCapture singleton
+	 * Also resets the view (supposed to be called once per page anyway)
 	 *
 	 * @param {object} config
 	 */
-	create: (config) => {
+	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_view = null;
 	},
 
 	/**
 	 * Creates a new View instance.
 	 *
 	 * @param {object} zoneConfigs
 	 */
-	startView: (zoneConfigs) => {
-		_view = new View({
+	startView: zoneConfigs => {
+		const viewProps = {
 			onMark: _onMark,
 			onMeasure: _onMeasure,
 			zoneConfigs,
-		});
+		};
+
+		// if we never had views before, it means we are in page view mode
+		if (_view) {
+			_view = new InteractiveView(viewProps);
+		} else {
+			_view = new PageView(viewProps);
+		}
```

Comment:
Wondering if this is better written as:

```
const views = { PageView, InteractiveView };
_view = new views[_view ? 'InteractiveView' : 'PageView'](viewProps);
```

Reply:
I toyed with it a bit, but then thought that this if with a comment would be best way documenting the logic.